qcdb/qcvars/glossary.py

```diff
@@ -987,6 +987,66 @@ def define_dashd_qcvars(fctl, dashes, extra=''):
 
 """}
 
+qcvardefs['CCSDT (PBE) TOTAL ENERGY'] = {
+    'units': 'Eh',
+    'glossary': r"""
+   The total electronic energy 
+   for the coupled cluster singles, doubles, and triples level of theory.
+"""}
+
+qcvardefs['CCSDT (PBE) CORRELATION ENERGY'] = {
+    'units': 'Eh',
+    'glossary': r"""
+   The correlation energy component
+   for the coupled cluster singles, doubles, and triples level of theory.
```

Comment:
there's going to be one of these for every functional, right? (e.g., `'CCSDT (BLYP) CORRELATION ENERGY'`). Not sure if we want to have all those defined. Also, the glossary line isn't unique from HF-reference `CCSDT CORRELATION ENERGY`